README.md

```diff
@@ -297,24 +303,10 @@ You can combine all priority but keep in mind that use `ORDER` and `POINTS_ASCEN
 | `emoji`         	| bool            	| For Windows is False else True 	                                  | On Windows, we have a problem printing emoji. Set to false if you have a problem      	                                                                                                  |
 | `colored`         | bool            	| True 	                                                              | If you want to print colored text [#45](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/45) [#82](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/82) |
 | `color_palette`   | ColorPalette      | All messages are Fore.RESET except WIN and LOSE bet (GREEN and RED) | Create your custom color palette. Read more above.      	                                                                                                                              |
+| `telegram`        | Telegram          | None                                                                | Receive Telegram updates for multiple events list [#233](https://github.com/Tkd-Alex/Twitch-Channel-Points-Miner-v2/issues/233)                                                           |
```

Comment:
I feel like it would be nice to indicate somewhere (other than just the default value here) that this can be omitted / set to `None` for people that doesn't want Telegram integration.

Most people will probably only read the big example above and to me it kinda feels like we're forced to obtain a token etc to have a valid conf.